### PR TITLE
Initial command and class skeleton

### PR DESCRIPTION
**Pull request by happz**

### CODE REVIEW

File: newa/__init__.py [hunk 1]
```diff
@@ -1,6 +1,132 @@
...
+
+
+T = TypeVar('T')
+SerializableT = TypeVar('SerializableT', bound='Serializable')
+
+
+def yaml_parser() -> ruamel.yaml.YAML:
+    """ Create standardized YAML parser """
+
+    yaml = ruamel.yaml.YAML()
```
Comment (lukaszachy) [inline newa/__init__.py:25]: Should we play safe?
```suggestion
    yaml = ruamel.yaml.YAML(typ='safe')
```

Comment (happz) [reply newa/__init__.py]: Fixed in 4d34e7f

**Comment by kkaarreell** [comment]
I think the `event` processor should be using different filename prefix for input (containing `[event]`) and produced files (containing also `[errata]`).

**Review by kkaarreell — APPROVED** [review]
The current version LGTM.  Thank you.

**Comment by kkaarreell** [comment]
> I think the `event` processor should be using different filename prefix for input (containing `[event]`) and produced files (containing also `[errata]`).



> I think the `event` processor should be using different filename prefix for input (containing `[event]`) and produced files (containing also `[errata]`).



> I think the `event` processor should be using different filename prefix for input (containing `[event]`) and produced files (containing also `[errata]`).

This has been addressed by introducing`init-` prefix for input files.

**Review by thrix — APPROVED** [review]
LGTM

**Pull request merged**